### ISSUE TITLE
Chip "Screening file missing"

### ISSUE THREAD

**Issue by 2amarkina** [blockframes/blockframes#7537]
We would like to help a user to get easier to the screening upload page in the Movie Tunnel during the event creation. 

To do that we want to introduce a click to the "Screening file is missing" chip. 

For the present moment we have only input chips that allow the interaction (deletion). 
[see the notion article here](https://www.notion.so/cascade8/Chips-Tags-95c6b208fa7d415d982ded1f220c5c3a)

TO DO: 
- [ ] introduce a new clickable chip (UI level, some UX discussions can be done)
- [ ] use this clickable chip for "Screening file is missing" 
- [ ] add a link to the Video section in Movie Tunnel, anchor the Screener section

**Comment by stephanie-cascade8** [comment]
We cannot use a chip there if we want consistensy and respect of UI good practices. 
Indeed a chip must not be clickable (it can be closed or be dragged and dropped but if it’s behaviour is a link then it becomes a button).

To my opinion, a good solution would be making a component out of something that already has the same behaviour and purpose in our app: the link on missing info in the summary page. 

It would give this result : 
[figma](https://www.figma.com/file/xigwniq5ZYfpD80gzQWYvO/Event?node-id=0%3A1)

<img width="764" alt="Capture d’écran 2022-01-26 à 10 35 24" src="https://user-images.githubusercontent.com/66258851/151138761-026575b6-f44b-4fff-a218-a2039689af30.png">

Here in the summary, I would like to make it the same component without icon and have it underlined since it's a link (for consistency and affordance).
<img width="764" alt="Capture d’écran 2022-01-26 à 10 35 24" src="https://user-images.githubusercontent.com/66258851/151138920-f3fdc263-35f8-413a-b580-0ee39536dc80.png">

**Comment by 2amarkina** [comment]
I like it !
The inconsistency that can happen is the following - on this page we can have two situations: 
1. When there is no title selected (now we have a chip which is not clickable)
<img width="1581" alt="Снимок экрана 2022-01-26 в 10 42 57" src="https://user-images.githubusercontent.com/55499036/151139977-e7d58bb3-638b-4560-ba7d-29c57aa8a26d.png">
2.  When the title is selected, but there is no screening file (a clickable action), so with link proposition it will become different.

And at the same time both infos will appear on the toolbar of the event and both info should be considered as warnings.
 
**Solutions / ideas:** 
- Maybe we could make "No title selected" also clickable and it will anchor to "select the title to display" field? 
Like that we would be consistent visually and in behavior too. 

@stephanie-cascade8 @cams-goussale @Jackseed what do you think?

**Comment by stephanie-cascade8** [comment]
> I like it ! The inconsistency that can happen is the following - on this page we can have two situations:
> 
> 1. When there is no title selected (now we have a chip which is not clickable)
> 
> <img alt="Снимок экрана 2022-01-26 в 10 42 57" width="1581" src="https://user-images.githubusercontent.com/55499036/151139977-e7d58bb3-638b-4560-ba7d-29c57aa8a26d.png">
> 
> 2. When the title is selected, but there is no screening file (a clickable action), so with link proposition it will become different.
> And at the same time both infos will appear on the toolbar of the event and both info should be considered as warnings.
> 
> **Solutions / ideas:**
> 
> * Maybe we could make "No title selected" also clickable and it will anchor to "select the title to display" field?
>   Like that we would be consistent visually and in behavior too.
> 
> @stephanie-cascade8 @cams-goussale @Jackseed what do you think?

Yes that would be a logical behaviour to me ! As long as we don't use chip like a link/button I'm ok 🙂 👍
